### PR TITLE
VB-570: Add mocked endpoints and first page check

### PR DESCRIPTION
**Pull request by psoleckimoj**
NOTE: These are likely to change as the tests are written, I just wanted a PR to get all the mocks in place.

### CODE REVIEW

File: integration_tests/mockApis/offenderSearch.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import { stubFor } from './wiremock'
+
+export default {
+  stubGetPrisoners: (): SuperAgentRequest => {
+    const results = {
+      totalPages: 1,
+      totalElements: 2,
+      content: [
+        {
+          lastName: 'test',
```
Comment (tpmcgowan) [inline integration_tests/mockApis/offenderSearch.ts:11]: Might be worth using distinct first/last names in the dummy data to be sure we're getting the them the right way round when they're outputted as `SURNAME, FIRSTNAME`!

Comment (psoleckimoj) [reply integration_tests/mockApis/offenderSearch.ts:11]: Will do when I get onto using them, just ripped them from existing tests.

File: integration_tests/pages/searchForAPrisoner.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import Page, { PageElement } from './page'
+
+export default class SearchForAPrisonerPage extends Page {
+  constructor() {
+    super('Search for a prisoner')
+  }
+
+  backLink = (): PageElement => cy.get('[href="/"]')
```
[the remark this answers is not in the document]
Comment (psoleckimoj) [reply integration_tests/pages/searchForAPrisoner.ts]: @tpmcgowan done!

File: integration_tests/mockApis/offenderSearch.ts (hunk 1 above)
Comment (tpmcgowan) [reply integration_tests/mockApis/offenderSearch.ts:11]: 👍

**Review by tpmcgowan — APPROVED** [review]
LGTM!

**Pull request merged**